### PR TITLE
Azure pipeline support for arm64 using qemu

### PR DESCRIPTION
**Pull request by odidev**
Added azure pipeline support for arm64 builds using qemu.
Rework for review [comment](https://github.com/aio-libs/multidict/pull/519#pullrequestreview-495544375).
@webknjaz Could you please review the changes?

### CODE REVIEW

File: templates/stage-deploy-aarch64.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    displayName: 'Manylinux_aarch64'
+    strategy:
+      matrix:
+        Py36 Arm64:
+          python.code: 'cp36-cp36m'
+          manylinux: 'manylinux_aarch64'
+        Py37 Arm64:
+          python.code: 'cp37-cp37m'
+          manylinux: 'manylinux_aarch64'
+        Py38 Arm64:
```
Comment (webknjaz) [inline templates/stage-deploy-aarch64.yml:15]: Does this image have CPython 3.9 in it?

File: templates/stage-deploy-aarch64.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+stages:
```
Comment (webknjaz) [inline templates/stage-deploy-aarch64.yml:1]: Looks like it's just a hardcoded pipeline. It's a template repo and it should contain only parametrized bits while the repos that use the templates would specify those params, like CPython versions.

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Comment by asvetlov** [comment]
Sorry, what project do you want to use the PR for? Yarl is already built with aarch64 support, isn't it?

**Comment by odidev** [comment]
@asvetlov I had raised this PR to add wheel support for aarch64 as per review comment [#discussion_r472823287](https://github.com/aio-libs/multidict/pull/519#discussion_r472823287). But this is not required now as multidict and yarl are using github actions for building wheels. So, closing this PR. Thanks.

**Pull request closed without merging**